docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -556,3 +564,42 @@ alias q=quarkus
 # Add q to list of commands included in quarkus autocompletion
 complete -F _complete_quarkus q
 ----
+
+
+=== Container images
+
+The quarkus CLI allows building container images without tampering with your project configuration (adding / removing container image extensions).
```

Comment:
```suggestion
The Quarkus CLI allows building container images without tampering with your project configuration (adding / removing container image extensions).
```

Reply:
I got the one but not the other?! EEEK. LOL